packages/nuxt/src/app/components/nuxt-root.vue

```diff
@@ -13,6 +13,7 @@
       :is="SingleRenderer"
       v-else-if="SingleRenderer"
     />
+    <NuxtPage v-else-if="route?.meta?.isolate" />
```

Comment:
It's probably better to use the lazy one

Reply:
Do you mean to use `LazyNuxtPage` instead? Doing this breaks a few more tests on the bundle:

```bash
Error: Snapshot `minimal nuxt application > default client bundle size 1` mismatched

Expected: ""116k""
Received: ""119k""
```

Is this kind of thing expected? Should I simply update these tests?

Reply:
For that test, just push and the CI will simply fix it for you.